### PR TITLE
Feature/get timesheet set

### PR DESCRIPTION
**Pull request by Nikita-Mamaisur**
Add TimesheetSet.Get.
Returns user timesheets for a certain day.

### CODE REVIEW

File: src/TimesheetSet.Get/TimesheetSet.Get.Api/Json/ApiNames.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+{
+    public const string DateFieldName = "gg_date";
+
+    public const string DurationFieldName = "gg_duration";
+
+    public const string HackproFieldName = "gg_hackpro";
+
+    public const string DescriptionFieldName = "gg_description";
+
+    public static readonly IReadOnlyCollection<string> SelectedFields = new[]
```
Comment (Sauric) [inline src/TimesheetSet.Get/TimesheetSet.Get.Api/Json/ApiNames.cs:16]: Статические поля лучше инициализировать через статический конструктор

**Review by Sauric — APPROVED** [review]

File: src/TimesheetSet.Get/TimesheetSet.Get.Api.Test/Test.TimesheetSetGet/Test.Invoke.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
+using GGroupp.Infra;
+using Moq;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace GGroupp.Internal.Timesheet.TimesheetSet.Get.Api.Test.Test;
```
Comment (pmosk) [inline src/TimesheetSet.Get/TimesheetSet.Get.Api.Test/Test.TimesheetSetGet/Test.Invoke.cs:8]: Namespace must be GGroupp.Internal.Timesheet.TimesheetSet.Get.Api.Test

File: src/TimesheetSet.Get/TimesheetSet.Get.Api.Test/TimesheetSet.Get.Api.Test.csproj [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	    <ImplicitUsings>disable</ImplicitUsings>
+	    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+	    <InvariantGlobalization>true</InvariantGlobalization>
+	    <IsPackable>false</IsPackable>
+	    <RootNamespace>GGroupp.Internal.Timesheet.TimesheetSet.Get.Api.Test</RootNamespace>
+	    <AssemblyName>GGroupp.Internal.Timesheet.TimesheetSet.Get.Api.Test</AssemblyName>
+    </PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="DeepEqual" Version="2.0.0" />
```
Comment (pmosk) [inline src/TimesheetSet.Get/TimesheetSet.Get.Api.Test/TimesheetSet.Get.Api.Test.csproj:15]: You should remove DeepEqual dependency. It is not used in this project

**Review by pmosk — CHANGES_REQUESTED** [review]

File: src/TimesheetSet.Get/TimesheetSet.Get.Api/Json/ApiNames.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace GGroupp.Internal.Timesheet;
+
+internal static class ApiNames
+{
+    public const string DateFieldName = "gg_date";
+
+    public const string DurationFieldName = "gg_duration";
+
+    public const string HackproFieldName = "gg_hackpro";
```
Comment (iShtopoR) [inline src/TimesheetSet.Get/TimesheetSet.Get.Api/Json/ApiNames.cs:12]: public const string ProjectFieldName = "gg_hackpro";

File: src/TimesheetSet.Get/TimesheetSet.Get.Api.Test/Test.TimesheetSetGet/Test.Invoke.cs (hunk 2 above)
Comment (Nikita-Mamaisur) [reply src/TimesheetSet.Get/TimesheetSet.Get.Api.Test/Test.TimesheetSetGet/Test.Invoke.cs]: fixed

File: src/TimesheetSet.Get/TimesheetSet.Get.Api.Test/TimesheetSet.Get.Api.Test.csproj (hunk 3 above)
Comment (Nikita-Mamaisur) [reply src/TimesheetSet.Get/TimesheetSet.Get.Api.Test/TimesheetSet.Get.Api.Test.csproj]: removed

File: src/TimesheetSet.Get/TimesheetSet.Get.Api/Json/ApiNames.cs (hunk 4 above)
Comment (Nikita-Mamaisur) [reply src/TimesheetSet.Get/TimesheetSet.Get.Api/Json/ApiNames.cs]: changed

**Review by iShtopoR — CHANGES_REQUESTED** [review]
.

**Review by pmosk — APPROVED** [review]

**Review by iShtopoR — APPROVED** [review]

**Pull request merged**